insPackages.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
```

Comment:
@eklee15: I think #2548 will help with this (i.e. we'll be able to default to `cmake%gcc`).  @scheibelp is working on it.